rdwatch/scoring/views/model_run.py

```diff
@@ -697,6 +697,7 @@ def get_sites_query(model_run_id: UUID4):
         sites=JSONBAgg(
             JSONObject(
                 id='pk',
+                site_id='site_id',
```

Comment:
Is this setting this to the literal string `'site_id'` or is it automatically doing `F('site_id')`?

Reply:
It's automatically doing `F('site_id')`. I thought the same thing when I saw `id='pk'`.